deploy/helm/README.md

```diff
@@ -110,9 +110,11 @@ The values of the `OPERATOR_NAMESPACE` and `OPERATOR_TARGET_NAMESPACES` determin
-| `trivy.tag`                              | `0.36.0`                           | Version of the Trivy image                                                                                                                                                                                                                                                                                                                                                                       |
-| `trivy.imagePullSecret`                  | N/A                                | imagePullSecret is the secret name to be used when pulling trivy image from private registries example: `reg-secret`. It is the user responsibility to create the secret for the private registry in `trivy-operator` namespace.                                                                                                                                                                 |
+| `trivy.image.registry`                         | `ghcr.io`                          | Registry of the Trivy image                                                                                                                                                                                                                                                                                                                                                                    |
+---------------------|
+| `trivy.image.repository`                       | `aquasecurity/trivy`               | Repository of the Trivy image                                                                                                                                                                                                                                                                                                                                                                    |
+| `trivy.image.tag`                              | `0.36.0`                           | Version of the Trivy image                                                                                                                                                                                                                                                                                                                                                                       |
+| `trivy.image.imagePullSecret`                  | N/A                                | imagePullSecret is the secret name to be used when pulling trivy image from private registries example: `reg-secret`. It is the user responsibility to create the secret for the private registry in `trivy-operator` namespace.                                                                                                                                                                 |
```

Comment:
these doc represent trivy config map and you changes impact only help setting the `cm` is not changing correct ?

Reply:
Yes, only the helm values are being changed here. The ConfigMap itself would still be the same.

Reply:
Oh, yes! Now I got your point. Will revert that one 😄